configure.go

```diff
@@ -0,0 +1,80 @@
+package discovery
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	kv "gopkg.in/clever/kayvee-go.v2"
+)
+
+// m is a convenience type for using kv.
+type m map[string]interface{}
+
+const (
+	templateVar = "SERVICE_%s_%s_%%s"
+)
+
+func getVar(envVar string) (string, error) {
+	envVar = strings.ToUpper(envVar)
+	val := os.Getenv(envVar)
+	if val == "" {
+		return "", errors.New(kv.FormatLog("discovery-go", kv.Error, "missing env var", m{
+			"var": envVar,
+		}))
+	}
+	return val, nil
+}
+
+// URL finds the specified URL for a service based off of the service's name and which
+// interface you are accessing. Values are found in environment variables fitting the scheme:
+// SERVICE_{SERVICE NAME}_{INTERFACE NAME}_{PROTO,HOST,PORT}.
+func URL(service, name string) (string, error) {
```

Comment:
sorry for being late to this PR, but i would curious what the INTERFACE_NAME is for. Is SERVICE_NAME not enough?